app/uk/gov/hmrc/cardpaymentfrontend/models/openbanking/OriginSpecificSessionData.scala

```diff
@@ -326,7 +328,7 @@ final case class WcClass1aNiSessionData(
     returnUrl:            Option[Url]          = None
 ) extends PayeSessionData(WcClass1aNi) {
   def paymentReference: Reference = ReferenceMaker.makeWcClass1aNiReference(wcClass1aNiReference)
-  def searchTag: SearchTag = SearchTag(wcClass1aNiReference.canonicalizedValue)
+  def searchTag: SearchTag = SearchTag(wcClass1aNiReference.canonicalizedValue.take(13))
```

Comment:
updating to just be the AOR, otherwise the open-banking payment won't be returned when web-chat lookups the payment